js/shopping-list-item.js

```diff
@@ -0,0 +1,17 @@
+class ShoppingListItem {
+  constructor(name, description) {
+    this.name = name;
+    this.description = description;
+    this.is_done = false;
+  }
+  check() {
+    this.is_done = true;
+  }
+  uncheck() {
+    this.is_done = false;
+  }
+  render() {
+    return `<li class= ${this.is_done}>
```

Comment:
The requirements for ShoppingListItem show the rendered element could have either `class="completed_true"` or `class="completed_false"`.
Currently it would output `class= true` or `class= false`.